internal/controller/standalone_pgadmin/pod.go

```diff
@@ -319,7 +319,8 @@ func startupCommand() []string {
 		ldapPasswordAbsolutePath = configMountPath + "/" + ldapFilePath
 
 		configSystem = `
-import json, re, os
+import glob, json, re, os
+DEFAULT_BINARY_PATHS = {'pg': sorted([''] + glob.glob('/usr/pgsql-*/bin')).pop()}
```

Comment:
I remember purposely leaving this out when I first needed to get this image working and just want to check: we need this because some pgAdmin function calls one of the PG binaries?

Reply:
Yeah the GUI backup uses pg_dump.